scripts/e2e.npm.publish.sh

```diff
@@ -46,11 +46,10 @@ npm-auth-to-token \
   -r http://localhost:4873
 
 # Prep branch for Lerna's git-checks
-BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
-if [ -z "$BRANCH" ]; then
-
-  BRANCH=$TRAVIS_BRANCH
-
+if [[ $GITHUB_REF = 'refs/pull/'* ]]; then
+  BRANCH=${GITHUB_HEAD_REF##*/}
```

Comment:
Stumbled upon a small problem with this pattern matcher during #2... 

If the branch name has a slash in it - e.g `issue/3112`, `BRANCH` will be set to `3112` and not get found.

Reply:
Ah good catch, ok I’ll have to dig into some more bash syntax to learn how to extract it properly (* after 2nd slash?)
Let me know if you have a suggestion.

Reply:
Peeked at the env vars [in CI][1] and it seemed like `GITHUB_HEAD_REF` *is* the extracted branch name. Maybe you just get this for free?

[1]: https://github.com/cgewecke/web3.js/pull/3/files
I tried 
  + PR into local branch
  + PR from fork
.. and both were routed through this `refs/pull/` clause. Is the other clause for push builds?

Reply:
Interesting, I was under the impression that head ref was only populated for PRs from forks, but if what you said is true then that’s great and makes it easy :)

Reply:
@ryanio 

> head ref was only populated for PRs from forks

Yes, that's what it says in the GH Actions docs...mmm.

Reply:
Attempted fix in https://github.com/ethereum/web3.js/pull/3468/commits/7297b5341f943de74cb0c5d68d57cf3664747974